script.rb

```diff
@@ -454066,7 +454066,7 @@ def raw_apply(snr)
 s.ins 0xc3, 4, 5, byte(1), -4
 s.ins 0x4f, :addr_0x92ce5, [4, Register.new(21)]
 s.ins 0x4f, :addr_0x926e2, [4, 0]
-s.ins 0x86, ushort(5953), byte(1), byte(1), s.layout('@rそしてそれは右代宮家へ、')
+s.ins 0x86, ushort(5953), byte(1), byte(1), s.layout('@rHeading towards the Ushiromiya manor,')
```

Comment:
Merge this with the next line (move the text from the next line over to this one, and put a `#` at the very start of the next line so that it gets ignored).